crates/cli/src/subcommands/generate/rust.rs

```diff
@@ -181,6 +180,7 @@ fn print_spacetimedb_imports(output: &mut Indenter) {
 
 fn print_file_header(output: &mut Indenter) {
     print_auto_generated_file_comment(output);
+    print_lines(output, &["#![allow(unused_imports)]"]);
```

Comment:
This is because rustc now complains about the unused `pub use foo_bar_baz_reducer::*;` glob import.